tiatoolbox/wsicore/wsireader.py

```diff
@@ -616,6 +682,47 @@ def slide_thumbnail(self, resolution=1.25, units="power"):
         thumb = self.read_bounds(bounds, resolution=resolution, units=units)
         return thumb
 
+    def tissue_mask(
+        self, method="otsu", resolution=1.25, units="power", **masker_kwargs
+    ):
+        """Create a tissue mask and wrap it in a VirtualWSIReader.
+
+        For the morphological method, mpp is used for calculating the
+        scale of the morphological operations. If no mpp is available,
+        objective power is used instead to estimate a good scale.
+        This can be overridden with a custom size, via passing a `kernel_size`
+        key-word argument in `masker_kwargs`, see
+        :class:`tissuemask.MorphologicalMasker` for more.
+
+
+        Args:
+            method (str): Method to use for creating the mask. Defaults
+                to 'otsu'. Methods are: otsu, morphological.
+            resolution (float): Resolution to produce the mask at.
+                Defaults to 1.25.
+            units (str): Units of resolution. Defaults to 'power'.
+            **masker_kwargs: Extra kwargs passed to the masker class.
+        """
```

Comment:
Please add a new line at the end.

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/61/commits/5b0e0ecc1fcb9ce989d907483bef9a2b0b2c7aa5